features/admin/remove_services.feature

```diff
@@ -0,0 +1,29 @@
+@remove-a-service
+Feature: Remove a service
+
+Scenario: Only Service Manager can remove services
+  Given I am logged in as the production admin-ccs-category user
+  # TODO get me here from the search
```

Comment:
Maybe for this we just make a new admin step (i.e. in `admin_steps.rb`) - something like `And I am on a random g-cloud service page` - that calls `I have a random g-cloud service from the api` internally and navigates to the page?